tests/foreman/ui/test_computeresource_ec2.py

```diff
@@ -53,108 +42,6 @@ def setUpClass(cls):
         cls.aws_security_groups = settings.ec2.security_groups
         cls.aws_managed_ip = settings.ec2.managed_ip
 
-    @run_only_on('sat')
-    @tier1
```

Comment:
not sure whether it is good idea to remove tier1 test cases as they can be covered separately for upstream?